tools/NodeConfigCompare/bootstrap_scripts/replay_bootstrap.sh

```diff
@@ -31,24 +31,20 @@ git clone https://github.com/aws/redshift-test-drive.git
 cd redshift-test-drive
 make setup
 if [[ "$SIMPLE_REPLAY_OVERWRITE_S3_PATH" != "N/A" ]]; then
-  aws s3 cp $SIMPLE_REPLAY_OVERWRITE_S3_PATH replay.yaml
+  aws s3 cp $SIMPLE_REPLAY_OVERWRITE_S3_PATH config/replay.yaml
 fi
 
 sed -i "s#master_username: \".*\"#master_username: \"$REDSHIFT_USER_NAME\"#g" config/replay.yaml
-sed -i "s#unload_iam_role: \".*\"#unload_iam_role: \"$REDSHIFT_IAM_ROLE\"#g" config/replay.yaml
 sed -i "s#workload_location: \".*\"#workload_location: \"$WORKLOAD_LOCATION\"#g" config/replay.yaml
 sed -i "s#target_cluster_endpoint: \".*\"#target_cluster_endpoint: \"$CLUSTER_ENDPOINT\"#g" config/replay.yaml
 sed -i "s#target_cluster_region: \".*\"#target_cluster_region: \"$TARGET_CLUSTER_REGION\"#g" config/replay.yaml
 
-if [[ "$SIMPLE_REPLAY_UNLOAD_STATEMENTS" == "true" ]]; then
-    sed -i "s#unload_iam_role: \".*\"#unload_iam_role: \"$REDSHIFT_IAM_ROLE\"#g" config/replay.yaml
-    sed -i "s#replay_output: \".*\"#replay_output: \"s3://$BUCKET_NAME/$REPLAY_PREFIX/$WHAT_IF_TIMESTAMP/$CLUSTER_IDENTIFIER\"#g" config/replay.yaml
-fi
+sed -i "s#unload_iam_role: \".*\"#unload_iam_role: \"$REDSHIFT_IAM_ROLE\"#g" config/replay.yaml
+sed -i "s#replay_output: \".*\"#replay_output: \"s3://$BUCKET_NAME/$REPLAY_PREFIX/$WHAT_IF_TIMESTAMP/$CLUSTER_IDENTIFIER\"#g" config/replay.yaml
```

Comment:
Wouldn't this cause unintended effects? Unload statements are more state changing than enabling Analysis by default.

Reply:
yes it would actually. 
I will comment the unload part